app/views/giraffe/postPayment.scala.html

```diff
@@ -19,7 +19,8 @@
                     <input id="marketingOptIn" type="checkbox" name="marketingOptIn" value="true"/>
                     <!--THIS COPY NEEDS TO CHANGE BEFORE MERGING -->
                     <label for="marketingOptIn">
-                        Whether you’re a subscriber, a member or you support us via a regular or one-off contribution, opt in here so we can keep sending you news, updates and more. Don’t worry, you can opt out at any time. </label>
+                        <strong>Subscriptions, membership and supporting The Guardian</strong> - Get related news and offers - whether you are a subscriber, member, supporter or would like to become on
```

Comment:
There's a typo here - on -> one

Reply:
What are we going to do without you Mike? 😕